thirdparty/github.go

```diff
@@ -262,18 +297,34 @@ func GetInstallationToken(ctx context.Context, owner, repo string, opts *github.
 			"owner":   owner,
 			"repo":    repo,
 		}))
-		return "", errors.Wrap(err, "creating installation token")
+		return legacyToken
 	}
-	return token, nil
+	if token == "" {
```

Comment:
we may want to add a log here for EVG-19966

Reply:
@bynn and I discussed logging. I pushed f1f9d70 to address his concerns, pending his review.